### PR TITLE
Clean up MapAdapter

### PR DESCRIPTION
**Pull request by scolsen**
This PR contains two minor clean up changes to the MapAdapter:

- Use timber for logging
- Remove the renderJsonLayer -- we don't need it anymore since we now take a viewport based approach to offline area selection﻿

### CODE REVIEW

File: gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java [hunk 1]
```diff
@@ -197,15 +197,15 @@ private static LatLng toLatLng(Point point) {
   }
 
   private void removeMarker(Marker marker) {
-    Log.v(TAG, "Removing marker " + marker.getId());
+    Timber.tag(TAG).v("Removing marker %s", marker.getId());
```
Comment (shobhitagarwal1612) [inline gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java]: We don't need to mention a custom tag. Timber uses the filename as the default tag. 

So, the above can be written as `Timber.v("Removing marker %s", marker.getId());`

File: gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java [hunk 2]
```diff
@@ -197,15 +197,15 @@ private static LatLng toLatLng(Point point) {
...
     marker.remove();
   }
 
   private int parseColor(@Nullable String colorHexCode) {
     try {
       return Color.parseColor(String.valueOf(colorHexCode));
     } catch (IllegalArgumentException e) {
-      Log.w(TAG, "Invalid color code in layer style: " + colorHexCode);
+      Timber.tag(TAG).w("Invalid color code in layer style: %s", colorHexCode);
```
Comment (shobhitagarwal1612) [inline gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java]: Same as above

File: gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java (hunk 1 above)
Comment (shobhitagarwal1612) [reply gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java]: This also makes `TAG` variable unused. So, please remove that as well

Comment (scolsen) [reply gnd/src/main/java/com/google/android/gnd/ui/map/gms/GoogleMapsMapAdapter.java]: Gotcha, done! 513ef9a

**Comment by scolsen** [comment]
Thanks @shobhitagarwal1612! It's ready for another check when you get a chance.

**Pull request merged**